bundles/org.openhab.binding.siemenshvac/src/main/resources/OH-INF/i18n/siemenshvac.properties

```diff
@@ -0,0 +1,28 @@
+# add-on
+
+addon.siemenshvac.name = SiemensHvac Binding
+addon.siemenshvac.description = This is the binding for SiemensHvac.
+
+# thing types
+
+thing-type.siemenshvac.ozw672.label = OZW672 IP Gateway
+thing-type.siemenshvac.ozw672.description = This is a OZW672 IP interface
+
+# thing types config
+
+thing-type.config.siemenshvac.ozw672.baseUrl.label = Base URL
+thing-type.config.siemenshvac.ozw672.baseUrl.description = The URL of the Siemens Hvac IP gateway. Must be in format http://hostname/ or https://hostname/. Don't forget the trailing '/'
+thing-type.config.siemenshvac.ozw672.userName.label = User Name
+thing-type.config.siemenshvac.ozw672.userName.description = User name of the Siemens Hvac gateway
+thing-type.config.siemenshvac.ozw672.userPassword.label = User Password
+thing-type.config.siemenshvac.ozw672.userPassword.description = User password of the Siemens Hvac gateway
+
+# offline message
+
+offline.waiting-bridge-initialization = Waiting bridge initialization, reading metadata in background
+
+offline.baseurl-mandatory = baseUrl is mandatory on configuration !
+offline.error-gateway-init = Error occurs during gateway initialization: %s
```

Comment:
```suggestion
offline.error-gateway-init = Error occurred during gateway initialization: {0}
```

Reply:
fix on next commit

Reply:
One space too much slipped in between the words "occurred" and "during".

Reply:
fix on next commit